README.md

```diff
@@ -5,6 +5,10 @@ linux: [![Build Status](https://travis-ci.org/ros-controls/ros2_control.svg?bran
 
 ## Disclaimer
 
+`ros2_control` is under heavy development, to get an update of the project please either tag along to one of our working group meetings (see more on discourse) and/or check the resources below:
+* Actively tracked projects can be found here: https://github.com/orgs/ros-controls/projects
+* Consult the roadmap here: https://github.com/ros-controls/roadmap
```

Comment:
I think you should add the demo repo, too. But you are more knowledgeable about the state of the demos.

* See demos here (under heavy development): https://github.com/ros-controls/ros2_control_demos

Reply:
good point, added!